holidays/locale/uk/LC_MESSAGES/IL.po

```diff
@@ -0,0 +1,93 @@
+# Israel holidays uk localization.
+# Authors: Arkadii Yakovets <ark@cho.red>, (c) 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Python Holidays 0.37\n"
+"POT-Creation-Date: 2023-07-18 17:31+0300\n"
+"PO-Revision-Date: 2023-07-18 17:57+0300\n"
+"Last-Translator: Arkadii Yakovets <ark@cho.red>\n"
+"Language-Team: Python Holidays localization team\n"
+"Language: uk\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingua 4.15.0\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#. Rosh Hashanah (New Year).
+msgid "ראש השנה"
+msgstr "Новий рік"
+
+#. Yom Kippur (Day of Atonement).
+msgid "יום כיפור"
+msgstr "Йом Кіпур"
+
+#. Sukkot (Feast of Tabernacles).
+msgid "סוכות"
+msgstr "Суккот"
+
+#. Simchat Torah / Shemini Atzeret.
+msgid "שמחת תורה/שמיני עצרת"
+msgstr "Симхат Тора / Восьмий день зборів"
```

Comment:
```suggestion
msgstr "Сімхат Тора / Восьмий день зборів"
```
In [Wiki](https://uk.wikipedia.org/wiki/%D0%A1%D1%96%D0%BC%D1%85%D0%B0%D1%82_%D0%A2%D0%BE%D1%80%D0%B0) and most online references in Ukrainian.